index.js

```diff
@@ -454,6 +454,7 @@ export const Classes = {
 		selected: 'selected'
 	},
 	alignments: {
+		alignment: 'alignment',
```

Comment:
just `alignment`? not `alignments`

Reply:
yeah, i'm using the `Rels.Alignments.alignments` to get to the alignments collection

This class is to get the individual alignments here

![image](https://user-images.githubusercontent.com/26232541/201168952-2b5b64e5-0e4d-48ab-a4b0-78bca5ba35b9.png)